moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -323,26 +337,35 @@ local function remove_objects(msg, doc, section, objects)
 end
 
 
+local function validate_schema(hsr, msg, doc, version)
```

Comment:
Built the default_schema in and documented its behavior; 'vacuous' can be removed from the schema repo